localstack-core/localstack/config.py

```diff
@@ -1032,6 +1032,13 @@ def populate_edge_configuration(
 # Whether to really publish to GCM while using SNS Platform Application (needs credentials)
 LEGACY_SNS_GCM_PUBLISHING = is_env_true("LEGACY_SNS_GCM_PUBLISHING")
 
+# Whether to enable the Next Gen APIGW invocation logic (handler chain)
+# TODO: remove this before merge!
+if not os.environ.get("PROVIDER_OVERRIDE_APIGATEWAY"):
+    os.environ["PROVIDER_OVERRIDE_APIGATEWAY"] = "next_gen"
```

Comment:
TODO remove before merge (as you already mentioned in the PR description, just another reminder 😛)